ArduPlane/tiltrotor.cpp

```diff
@@ -256,7 +256,7 @@ void QuadPlane::tilt_compensate_down(float *thrust, uint8_t num_motors)
     // control while angled over. This greatly improves the stability
     // of the last phase of transitions
     float tilt_threshold = (tilt.max_angle_deg/90.0f);
-    bool equal_thrust = (tilt.current_tilt > tilt_threshold);
+    bool equal_thrust = (tilt.current_tilt >= tilt_threshold);
```

Comment:
this doesn't look like a significant change; magnitude of the effect is a shift of FLT_EPSILON?

Reply:
the difference is that we sit at exactly tilt_threshold for as long as it takes to reach min airspeed. So it can sit at an == for quite a long time.

Reply:
duh, I see now. This is a setpoint, not a measurement.